src/integration-test/tests/citizen/paidInFull/paid_in_full_test.ts

```diff
@@ -1,21 +0,0 @@
-import I = CodeceptJS.I
```

Comment:
This change should probably be made as a separate PR?
its unrelated to this change

Reply:
this was a bit of a hack... master has been updated already

Reply:
I assume this file needs to be restored then?
This branch is up to date with master

Reply:
Can we include? :) This was so I could get a green build so then we can merge and start using AKS. The PR with this change is having some issues getting to preview env.

Reply:
master is currently broken, removing this file is the fix. waiting on a green build on the other PR before merging. this being merged would fix most of the fail deployments that have been delaying the merge of the other PR. happy for it to be merge in so long as there's a green build on the e2e tests

Reply:
aah cool, only reason I commented the above was because @markry-kainos said that the fix was on master